src/reader/reader.ts

```diff
@@ -41,7 +41,7 @@ function readList(firsttok: Tok.LPar, tokens: Token[], i: number): PartialReadRe
   const data3: SExpr[] = [];
 
   // Stage 1, no . seen yet
-  while (true) {
+  for (;;) {
```

Comment:
didn't know you could do this! Is this a code style change or is there a semantic difference with `while (true)`?

Reply:
eslint doesn't like constant conditions, so this is just another way of writing an infinite loop that doesn't have constant conditions